docs/docs/configuration/offline-readers.md

```diff
@@ -0,0 +1,12 @@
+# Offline Readers
```

Comment:
Originally we thought of calling this `offline users`, let's talk this through with the rest of the team.